AzureLoggingAlertsToTeamsWithStorage

```diff
@@ -0,0 +1,663 @@
+                        "body": {
+                          "messageBody": "<div style=\"border: 2px solid @{variables('AlertBorderColor')}; padding: 16px 32px; border-radius: 5px\"><div>\n<h2>\n@{triggerBody()?['data']?['essentials']?['alertRule']}</h2>\n<p></p>\n@{body('Get_an_@mention_token_for_a_user')?['atMention']}\n<p>Resourcelink: <a href=\"https://portal.azure.com/#@6380bae5-10b0-4942-a657-b66ab2b072ec/resource@{triggerBody()?['data']?['essentials']?['alertTargetIDs'][0]}/alertsV2\">Azure portal resource</a></p>\n<p><span style=\"white: space\">Severity:&nbsp;</span>@{triggerBody()?['data']?['essentials']?['severity']}</p>\n<p><span style=\"white: space\">Status:&nbsp;</span>@{triggerBody()?['data']?['essentials']?['monitorCondition']}</p>\n<p></p></div></div>",
+                          "parentMessageId": "@{body('Get_entity')?['MessageId']}",
+                          "recipient": {
+                            "channelId": "19:6Rw40CznKci4iFLtxz_WILuPD6pWJA80M89VJnYA4uU1@thread.tacv2",
+                            "groupId": "b0a24c92-35a5-47d5-ae07-20a21e772476"
+                          }
+                        },
+                        "host": {
+                          "connection": {
+                            "name": "@parameters('$connections')['teams']['connectionId']"
+                          }
+                        },
+                        "method": "post",
+                        "path": "/v1.0/teams/conversation/replyWithMessage/poster/Flow bot/location/@{encodeURIComponent('Channel')}"
+                      }
+                    },
+                    "Set_variable_7": {
+                      "runAfter": {},
+                      "type": "SetVariable",
+                      "inputs": {
+                        "name": "AlertBorderColor",
+                        "value": "#0bf707"
+                      }
+                    }
+                  }
+                }
+              },
+              "default": {
+                "actions": {}
+              },
+              "expression": "@{triggerBody()?['data']?['essentials']?['monitorCondition']}",
+              "type": "Switch"
+            }
+          },
+          "outputs": {}
+        },
+        "parameters": {
+          "$connections": {
+            "value": {
+              "azuretables": {
+                "connectionId": "[resourceId('Microsoft.Web/connections', parameters('connections_azuretables_name'))]",
+                "connectionName": "azuretables",
+                "id": "/subscriptions/06d56d01-0334-449a-8e17-1f46ade941d8/providers/Microsoft.Web/locations/westeurope/managedApis/azuretables"
+              },
+              "teams": {
+                "connectionId": "[parameters('connections_teams_externalid')]",
+                "connectionName": "teams",
+                "id": "/subscriptions/06d56d01-0334-449a-8e17-1f46ade941d8/providers/Microsoft.Web/locations/westeurope/managedApis/teams"
```

Comment:
Dit is nu ook fixed? (subscription id)

Reply:
Subscription id aangepast naar parameter